source/Octopus.Tentacle/Octopus.Tentacle.csproj

```diff
@@ -52,7 +52,7 @@
 		<PackageReference Include="KubernetesClient" Version="13.0.26" />
 	</ItemGroup>
 	<ItemGroup>
-		<PackageReference Include="Octopus.Client" Version="14.3.1436" />
+		<PackageReference Include="Octopus.Client" Version="14.3.1467-kevjt-kubernetes-agent-as-a-worker-20240624172217" />
```

Comment:
This will be updated once the draft PRs finalise and we get a proper release of the Octopus.Client changes